src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +205,38 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+		/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A password-field is intended for entering passwords.
+	 *   composition: >
+	 *      Text password will render an input-tag with type="password".
+	 *   effect: >
+	 *      Text password is restricted to one line of text and will
+	 *      mask the entered characters.
+	 *   rivals:
+	 *      text field: >
+	 *          Use a text field for discloseable information (i.e.
+	 *          information that can safely be displayed to an audience)
+	 *
+	 * context: Login-Form and own profile (change Password).
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Password Input MUST be used for passwords.
+	 *   interaction:
+	 *     1: >
+	 *         Password Input MUST NOT limit the number of characters.
```

Comment:
`MUST NOT` -> `SHOULD NOT` - There might be cases, where a password is stored unencrypted or has specific requirements for length, such as when an LDAP-Password is saved.